yarn.lock

```diff
@@ -3382,14 +3387,14 @@ react-dom@^16.5:
     scheduler "^0.11.2"
 
 react@^16.5:
-  version "16.6.3"
-  resolved "https://registry.yarnpkg.com/react/-/react-16.6.3.tgz#25d77c91911d6bbdd23db41e70fb094cc1e0871c"
-  integrity sha512-zCvmH2vbEolgKxtqXL2wmGCUxUyNheYn/C+PD1YAjfxHC54+MhdruyhO7QieQrYsYeTxrn93PM2y0jRH1zEExw==
+  version "16.7.0"
```

Comment:
This looks like it got slightly broken. `react` and `react-dom` are now pointing to different versions, which also means different versions of `scheduler`.